tox.ini

```diff
@@ -75,6 +83,19 @@ deps =
     atheris==2.0.11
 setenv =
     {[testenv]setenv}
+commands = {[framework]commands}
+
+[testenv:py3.10-linux-packages]
```

Comment:
for consistency with `open-aea` please make `packages-` the prefix rather than suffix